src/main/java/profile_manager/frameworks_and_drivers/ProfileScreen.java

```diff
@@ -0,0 +1,134 @@
+package profile_manager.frameworks_and_drivers;
+
+import profile_manager.interface_adapters.ProfileManagerController;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.*;
+
+public class ProfileScreen extends JFrame implements ActionListener {
+
+    JTextField profileName = new JTextField(15);
+    JTextField dob = new JTextField(15);
+    JTextField description = new JTextField(15);
+    JTextField socialLinks = new JTextField(15);
+    JTextField sensitiveWords = new JTextField(15);
+    JTextField interests = new JTextField(15);
+    JTextField groups = new JTextField(15);
+    JTextField friends = new JTextField(15);
+    JTextField blockedUsers = new JTextField(15);
+    ProfileManagerController profileManagerController;
+    String loginUserName;
+
+
+    /**
+     * A window with a title and a JButton.
+     */
+    public ProfileScreen(ProfileManagerController controller, String  loginUserName) {
+
+        this.profileManagerController = controller;
+        this.loginUserName = loginUserName;
+
+        JTextField userName = new JTextField(this.loginUserName,15);
+        userName.setEnabled(false);
+
+        JLabel title = new JLabel("Profile Screen");
+        title.setAlignmentX(Component.CENTER_ALIGNMENT);
+
+        LabelTextPanel userNameInfo = new LabelTextPanel(
+                new JLabel("UserName"), userName);
+        LabelTextPanel profileNameInfo = new LabelTextPanel(
+                new JLabel("ProfileName"), profileName);
+        LabelTextPanel dobInfo = new LabelTextPanel(
+                new JLabel("DOB"), dob);
+        LabelTextPanel descriptionInfo = new LabelTextPanel(
+                new JLabel("Description"), description);
+        LabelTextPanel socialLinksInfo = new LabelTextPanel(
+                new JLabel("SocialLinks"), socialLinks);
+        LabelTextPanel sensitiveWordsInfo = new LabelTextPanel(
+                new JLabel("SensitiveWords"), sensitiveWords);
+        LabelTextPanel interestsInfo = new LabelTextPanel(
+                new JLabel("Interests"), interests);
+        LabelTextPanel groupsInfo = new LabelTextPanel(
+                new JLabel("Groups"), groups);
+        LabelTextPanel friendsInfo = new LabelTextPanel(
+                new JLabel("Friends"), friends);
```

Comment:
I have a concern about the "friends" label here. I don't really understand why the friend list of somebody should appear on its own profile while other users can see it. Instead, I think it should appear in the user's own info.